diagnostic.md

```diff
@@ -51,13 +57,16 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+# curl --include --request DELETE http://www.example.com/donuts
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = $.ajax({
+  url: config.apiOrigin + '/donut',
```

Comment:
This has a similar issue as the previous question about requesting a single resource.